l1-contracts/contracts/bridge/asset-router/L2AssetRouter.sol

```diff
@@ -28,6 +28,9 @@ contract L2AssetRouter is AssetRouterBase, IL2AssetRouter {
     /// @dev The address of the L2 legacy shared bridge.
     address public immutable L2_LEGACY_SHARED_BRIDGE;
 
+    /// @dev The asset id of the base token.
+    bytes32 public immutable BASE_TOKEN_ASSET_ID;
```

Comment:
do you think it makes sense to store it on both L1 and L2 AR? or it wont be used. on L1 at all?

Reply:
I don't think we use it on the L1 atm, we use ETH_TOKEN_ADDRESS there